apps/tangle-dapp/utils/convertToSubstrateAddress.ts

```diff
@@ -1,25 +1,15 @@
-import {
-  encodeAddress,
-  evmToAddress,
-  isEthereumAddress,
-} from '@polkadot/util-crypto';
-
-const SS58_PREFIX = 5;
+import { evmToAddress, isEthereumAddress } from '@polkadot/util-crypto';
 
 /**
  * Converts an Ethereum address to a Substrate address.
  *
  * @param address - The address to be converted.
- * @param substratePrefix - The SS58 prefix for the Substrate network.
  * @returns The converted Substrate address
  */
-export const convertToSubstrateAddress = (
-  address: string,
-  substratePrefix: number = SS58_PREFIX
-): string => {
+export const convertToSubstrateAddress = (address: string): string => {
   if (isEthereumAddress(address)) {
-    return evmToAddress(address, substratePrefix);
+    return evmToAddress(address);
+  } else {
+    return 'invalid';
```

Comment:
Should we throw an error here instead OR `return null` (to force the caller to handle the possibility that this may fail), to avoid having the caller continue execution with an invalid address?